packages/backend/src/misc/is-renote.ts

```diff
@@ -36,6 +36,17 @@ export function isQuote(note: Renote): note is Quote {
 		note.fileIds.length > 0;
 }
 
```

Comment:
```suggestion

// 上の関数はrenoteIdを見ているためMiNoteをjoinしてなくてもtrueを返すが、この関数はrenoteをちゃんとjoinしていないとfalseが返ってくる
```

並べる場合は差がわかったほうがいいと思うのでコメントの追加を提案します。

Reply:
やっぱりここは共通化しない方が分かりやすそうな気がしたから戻しました

Reply:
こっちだけいじってもう片方は忘れるってのを絶対やるのでどっかしらにコメントは残しておきたい

上の`// NOTE: SYNC WITH NoteCreateService.isQuote` みたいに